src/azure-cli/azure/cli/command_modules/monitor/transformers.py

```diff
@@ -69,6 +69,16 @@ def row_convert(item):
     return _generic_table_convert(results, row_convert)
 
 
+def metrics_namespaces_table(results):
```

Comment:
Why do we need the transform?

Reply:
By default `metricNamespaceName` will not show as a table column because it's the second level property.